### PR TITLE
feat(app): add gripper requirement to protocol and run requirements 

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Everywhere in the app experience, where we define the requirements of a particular protocol now
includes the gripper if said protocol includes a move_labware command that uses the gripper.

[RLAB-154](https://opentrons.atlassian.net/browse/RLAB-154)

# Review requests

- upload a protocol that uses the gripper
- should show as requirement on:
   - protocol list
   - protocol detail > robot configuration
   - run setup > robot calibration

# Risk assessment
low

[RLAB-154]: https://opentrons.atlassian.net/browse/RLAB-154?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/organisms/Devices/ProtocolRun/ProtocolRunSetup.tsx [hunk 1]
```diff
@@ -180,6 +180,8 @@ export function ProtocolRunSetup({
     },
   }
 
+  console.log('runHasStarted', runHasStarted)
+  console.log('calibrationStatus', calibrationStatus)
```
Comment (smb2268) [inline app/src/organisms/Devices/ProtocolRun/ProtocolRunSetup.tsx]: Looks like these are leftover from testing

File: app/src/organisms/Devices/ProtocolRun/__tests__/SetupPipetteCalibration.test.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { i18n } from '../../../../i18n'
 import { mockTipRackDefinition } from '../../../../redux/custom-labware/__fixtures__'
 import { useRunPipetteInfoByMount } from '../../hooks'
 import { SetupPipetteCalibrationItem } from '../SetupPipetteCalibrationItem'
-import { SetupPipetteCalibration } from '../SetupPipetteCalibration'
+import { SetupPipetteCalibration } from '../SetupInstrumentCalibration'
```
Comment (smb2268) [inline app/src/organisms/Devices/ProtocolRun/__tests__/SetupPipetteCalibration.test.tsx]:
```suggestion
import { SetupInstrumentCalibration } from '../SetupInstrumentCalibration'
```

File: app/src/organisms/Devices/ProtocolRun/__tests__/SetupRobotCalibration.test.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ import {
 import { mockDeckCalData } from '../../../../redux/calibration/__fixtures__'
 import { useDeckCalibrationData, useIsOT3, useRunHasStarted } from '../../hooks'
 import { SetupDeckCalibration } from '../SetupDeckCalibration'
-import { SetupPipetteCalibration } from '../SetupPipetteCalibration'
+import { SetupPipetteCalibration } from '../SetupInstrumentCalibration'
```
Comment (smb2268) [inline app/src/organisms/Devices/ProtocolRun/__tests__/SetupRobotCalibration.test.tsx]:
```suggestion
import { SetupInstrumentCalibration } from '../SetupInstrumentCalibration'
```

File: app/src/organisms/Devices/RobotCard.tsx [hunk 4]
```diff
@@ -31,14 +37,13 @@ import { StyledText } from '../../atoms/text'
 import { CONNECTABLE, getRobotModelByName } from '../../redux/discovery'
 import { ModuleIcon } from '../../molecules/ModuleIcon'
 import { UpdateRobotBanner } from '../UpdateRobotBanner'
-import { useAttachedModules, useAttachedPipettes, useIsOT3 } from './hooks'
+import { useAttachedModules, useIsOT3 } from './hooks'
```
Comment (smb2268) [inline app/src/organisms/Devices/RobotCard.tsx]:
```suggestion
import { useIsOT3 } from './hooks'
```

File: app/src/pages/Devices/DevicesLanding/index.tsx [hunk 5]
```diff
@@ -58,6 +58,10 @@ export function DevicesLanding(): JSX.Element {
     reachableRobots,
     robot => robot.healthStatus === 'ok'
   )
+  console.log('healthyReachableRobots', healthyReachableRobots)
+  console.log('unhealthyReachableRobots', unhealthyReachableRobots)
+  console.log('reachableRobots', reachableRobots)
+  console.log('unreachableRobots', unreachableRobots)
```
Comment (smb2268) [inline app/src/pages/Devices/DevicesLanding/index.tsx]: We should remove these also

**Review by smb2268 — APPROVED** [review]
This looks good to me!

**Review by shlokamin — CHANGES_REQUESTED** [review]
added some fixes but the `run a protocol` button in the overflow menu does not work. bailing on this for now to get other stuff in 0.9.0

**Review by shlokamin — APPROVED** [review]
okay i think we should be good now

**Pull request merged**